src/test/resources/it/pagopa/pn/cucumber/workflowNotifica/b2b/pf/AvanzamentoNotificheAsyncB2bPF.feature

```diff
@@ -132,16 +132,18 @@ Feature: avanzamento notifiche asincrone b2b PF - controllo costi
       | pagoPaIntMode      | ASYNC                       |
       | paFee              | 10                           |
     And destinatario
-      | denomination          | Cristoforo Colombo |
-      | taxId                 | CLMCST42R12D969Z   |
-      | physicalAddress_zip   | KO                 |
-      | digitalDomicile       | NULL               |
-      | payment_creditorTaxId | 77777777777        |
-      | payment_pagoPaForm    | SI                 |
-      | payment_f24flatRate   | NULL               |
-      | payment_f24standard   | NULL               |
-      | apply_cost_pagopa     | SI                 |
-      | payment_multy_number  | 1                  |
+      | denomination                 | Cristoforo Colombo |
```

Comment:
And destinatario Mario Gherkin (o Mario Cucumber non ricordo) e: 
al posto di  
| denomination          | Cristoforo Colombo |
| taxId                 | CLMCST42R12D969Z   |